web_external/templates/body/tasksPage.jade

```diff
@@ -0,0 +1,15 @@
+.container.isic-tasks-container
```

Comment:
@msmolens Following with what our other views do, and to keep the proliferation of stylesheets under control, I think we can put all this in `taskPage.styl`, right?